ddtrace/tracer.py

```diff
@@ -224,9 +243,11 @@ def __init__(
         self._partial_flush_enabled = asbool(os.getenv("DD_TRACE_PARTIAL_FLUSH_ENABLED", default=True))
         self._partial_flush_min_spans = int(os.getenv("DD_TRACE_PARTIAL_FLUSH_MIN_SPANS", default=500))
         self._appsec_enabled = config._appsec_enabled
+        # Direct link to the appsec processor
+        self._appsec_processor = None
         self._iast_enabled = config._iast_enabled
 
-        self._span_processors = _default_span_processors_factory(
+        self._span_processors, self._appsec_processor = _default_span_processors_factory(
```

Comment:
is `self._appsec_processor` only used by tests?

Reply:
It's also used in `remoteconfiguration.py`.